components/map.jsx

```diff
@@ -70,6 +71,40 @@ var MarkerPopup = React.createClass({
   }
 });
 
+var ClubStatusLabel = React.createClass({
+  propTypes: {
+    status: React.PropTypes.string.isRequired,
+    showApproved: React.PropTypes.bool
+  },
+  render: function() {
+    if (this.props.status === 'pending') {
+      return (
+        <span className="label label-warning"
+         title="This club is pending approval and is not visible to other users.">
+          pending
```

Comment:
Since `this.props.status` is exactly the same as the actual text of the label, could we just use that variable here?  It might be my personal preference but in case we decide to change the text of the status (e.g., denied) in the future we then will have one fewer place in the code to visit.  :grin: